tests/utils/verification/volumeproperties.go

```diff
@@ -154,7 +154,7 @@ func VerifyDetachedStatus(name, hostName, esxName string) bool {
 			return true
 		}
 	}
-	log.Fatalf("Timed out to poll status\n")
+	log.Printf("Timed out to poll status\n")
```

Comment:
it fails to return boolean ... the next line anyway returns false and testcase handles it correctly by asserting on the return value on the other hand `Fatalf` is too harsh it stop the test run and exits. 

We should avoid using "Fatal" here.